specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/ManagedInstanceDtcs.json

```diff
@@ -269,6 +269,16 @@
         "snaLu6point2TransactionsEnabled": {
           "description": "Allow SNA LU 6.2 to managed instance DTC.",
           "type": "boolean"
+        },
+        "xaTransactionsDefaultTimeout": {
```

Comment:
>xaTransactionsDefaultTimeout [](http://example.com/codeflow?start=9&length=28)

I see the PR is marked as 'New API Version' but I'm not seeing an API version bump, can you update the change log? What will be the behavior for the existing user on older client to get a response that contains these two new field?

Reply:
I've updated the changelog in the PR description. Copying it here also:
This change, introduces 2 new properties in managed instance DTC security settings. Those 2 properties are optional integers - xaTransactionsDefaultTimeout and xaTransactionsMaximumTimeout. The version bump is not visible on this PR, since this is a dev branch in which all the controllers from the previous version are copied and have their API version bumped.
      The newly generated clients will be able to set the mentioned properties when updating the managed instance DTC, and those 2 properties will be visible when fetching the managed instance's DTC. Since they are optional, they can be omitted in the request - the underlying resource won't change/replace them if they are missing.